code/media/build.gradle

```diff
@@ -52,7 +52,7 @@ android {
 
 dependencies {
     //noinspection GradleDependency
-    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
+    implementation "com.adobe.marketing.mobile:core:$mavenCoreVersion"
     implementation 'androidx.annotation:annotation:1.3.0'
```

Comment:
aren't we using 1.0.0 everywhere?

Reply:
https://github.com/adobe/aepsdk-edge-android/blob/main/code/gradle.properties#L33
https://github.com/adobe/aepsdk-edge-android/blob/main/code/edge/build.gradle#L239